packages/swapper/src/swappers/ChainflipSwapper/endpoints.ts

```diff
@@ -148,79 +94,36 @@ export const chainflipApi: SwapperApi = {
       gasPrice: unsignedTxInput.gasPrice,
     }
   },
-  getUnsignedUtxoTransaction: async ({
+  getUnsignedUtxoTransaction: ({
     tradeQuote,
     senderAddress,
     xpub,
     accountType,
     assertGetUtxoChainAdapter,
-    config,
   }: GetUnsignedUtxoTransactionArgs): Promise<BTCSignTx> => {
     if (!isExecutableTradeQuote(tradeQuote)) throw Error('Unable to execute trade')
 
-    const brokerUrl = config.REACT_APP_CHAINFLIP_API_URL
-    const apiKey = config.REACT_APP_CHAINFLIP_API_KEY
-
     const step = tradeQuote.steps[0]
 
     if (!isExecutableTradeStep(step)) throw Error('Unable to execute step')
+    if (!step.chainflipSpecific?.chainflipDepositAddress) throw Error('Missing deposit address')
```

Comment:
q: do we want to do similar checks for `step.chainflipSpecific?.chainflipSwapId` and remove opt chain below?

Reply:
yes, good idea!